srcpkgs/gcc/template

```diff
@@ -19,15 +19,15 @@ license="GFDL-1.2-or-later, GPL-3.0-or-later, LGPL-2.1-or-later"
 # *-musl builders have issues fetching https://ftp.gnu.org/gnu/mpc/mpc-1.1.0.tar.gz
 distfiles="
  ${GNU_SITE}/gcc/gcc-${version}/gcc-${version}.tar.xz
- https://gmplib.org/download/gmp/gmp-${_gmp_version}.tar.xz
+ ${GNU_SITE}/download/gmp/gmp-${_gmp_version}.tar.xz
```

Comment:
`${GNU_SITE}/download` should just be `${GNU_SITE}`

Reply:
Thanks, I didn't notice sine the old source was still in my hostdir.